Xamarin.Forms.Platform.UAP/EditorRenderer.cs

```diff
@@ -106,6 +107,10 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateMaxLength();
 			else if (e.PropertyName == Specifics.DetectReadingOrderFromContentProperty.PropertyName)
 				UpdateDetectReadingOrderFromContent();
+			else if (e.PropertyName == InputView.IsReadOnlyProperty.PropertyName)
+			{
```

Comment:
Can skip adding curly braces if there is a single statement nested under a predicate.

Reply:
I will fix this :)

Reply:
I will :)

Reply:
this is fine